pkg/handler/config.go

```diff
@@ -122,6 +128,10 @@ func ReadConfigFile(version, date, filename string) (*Config, error) {
 			Filters:      []Filter{},
 			QuickFilters: []QuickFilter{},
 			Features:     []string{},
+			Deduper: Deduper{
```

Comment:
can u pls add a note here so when the agent default change we need to remember and change this as well ? OR it will be much better to have only one source of truth can u read it from the same cm instead of hard coding it here ?

Reply:
I've added a TODO for now. We could load by default the sample yaml but that would force us to embed it in the code and parse two yaml, one after the other.

The plugin accept as arguments a file path that is configurable. These defaults are overriden below (line 141)